pkg/processor/reportprocessor.go

```diff
@@ -320,7 +320,16 @@ func getSevFromTemplate(plc unstructured.Unstructured, name string) string {
 			return severity
 
 			// If this isn't an OCM policy, check for a severity annotation
-		} else if severityAnnotation, ok := plc.GetAnnotations()["policy.open-cluster-management.io/severity"]; ok {
+		} else {
+			severityAnnotation, _, err := unstructured.NestedString(
+				objDef, "metadata", "annotations", "policy.open-cluster-management.io/severity",
+			)
+			if err != nil {
```

Comment:
@dhaiducek Do we need to consider the possibility that the annotation is not found?
```suggestion
			severityAnnotation, severityFound, err := unstructured.NestedString(
				objDef, "metadata", "annotations", "policy.open-cluster-management.io/severity",
			)
			if !severityFound || err != nil {
```

Reply:
I could go either way on it since the default return value for the function is an empty string, but this would probably make more sense.